apps/test/integration/levelTests.js

```diff
@@ -110,7 +111,7 @@ describe('Level tests', function () {
   beforeEach(function () {
     // Recreate our redux store so that we have a fresh copy
     stubRedux();
-    registerReducers({stageLock});
+    registerReducers({stageLock, runState});
```

Comment:
runState reducer now needs to get added earlier because of line 147 of this file: https://github.com/code-dot-org/code-dot-org/pull/13307/files#diff-dd0a9add5d472bc3f7f5c1d91d652773R147, which depends on state that now got moved to redux.